packages/fxa-auth-server/lib/routes/oauth/index.js

```diff
@@ -291,9 +291,13 @@ module.exports = (log, config, oauthdb, db, mailer, devices) => {
             uid = tokenVerify.user;
           }
 
+          const account = await db.account(uid);
+          const ecosystemAnonId = account.ecosystemAnonId;
+
           await request.emitMetricsEvent('oauth.token.created', {
             grantType: request.payload.grant_type,
             uid,
+            ecosystemAnonId,
```

Comment:
@6a68 This would log out the entire id, which I was under the impression could be a very long string. Is there something inside the decoded value that we should log here instead?

Reply:
@vbudhram good question, the ecosystem_anon_id is encrypted and the private keys are held by the pipeline, so we have to log the whole thing